examples/verify-rollup/src/partial-handlers/rollup.ts

```diff
@@ -467,5 +468,34 @@ export const rollupHandler = async (
   }
   console.log('');
 
+  //
+  // STF verification
+  //
+
+  // get the wasmModuleRoot of the given orbit chain
+  const moduleRoot = (await orbitHandler.readContract(
+    'parent',
+    rollupAddress,
+    [...RollupCore__factory.abi, ...Ownable__factory.abi] as Abi,
+    'wasmModuleRoot',
+  )) as `0x${string}`;
+
+  const index = WASMModuleRoots.indexOf(moduleRoot);
+
+  // check if this wasmModuleRoot belongs to one of mainnet version
+  if (0 <= index) {
+    console.log('The state transition function is standard version');
+    // check if the rollups' arbos version is latest or not
+    if (index < currentMainnetWasmModuleRootIndex) {
+      warningMessages.push('Arbos version is old');
```

Comment:
Can you add both module roots here? Something like "... Rollup wasmModuleRoot is ${moduleRoot}. Latest standard wasmModuleRoot is ${WASMModuleRoots[currentMainnetWasmModuleRootIndex]}."